pkgs/development/libraries/kde-frameworks/bluez-qt.nix

```diff
@@ -14,6 +14,6 @@ mkDerivation {
   propagatedBuildInputs = [ qtbase ];
   preConfigure = ''
     substituteInPlace CMakeLists.txt \
-      --replace /lib/udev/rules.d "$bin/lib/udev/rules.d"
+      --replace /lib/udev/rules.d "$out/lib/udev/rules.d"
```

Comment:
Because `$bin` is present and the default output, this prevents the rules from being installed. Is that what we want?

Reply:
Not sure if I understand. https://github.com/NixOS/nixpkgs/pull/93006/files#diff-c1c886b16586c62e53e0d38c07f9bb6dR193 should pick udev rules from `lib` (which doesn't exist here), and otherwise fallback to `$out` - which is where we install now into.